src/kokkos_ext/ArborX_DetailsKokkosExtViewHelpers.hpp

```diff
@@ -36,10 +36,10 @@ lastElement(ExecutionSpace const &space, Kokkos::View<T, P...> const &v)
   auto const n = v.extent(0);
   ARBORX_ASSERT(n > 0);
   auto v_subview = Kokkos::subview(v, n - 1);
-  auto v_host = Kokkos::create_mirror_view(v_subview); // FIXME
+  typename Kokkos::ViewTraits<T, P...>::non_const_value_type v_host;
   Kokkos::deep_copy(space, v_host, v_subview);
```

Comment:
Since when does Kokkos support a copy into a POD on the host?

Reply:
Since always for rank-0 views.